frontend/vre/main.js

```diff
@@ -27,6 +27,7 @@ import { WelcomeView } from './utils/welcome.view.js';
 GlobalVariables.myCollections = new VRECollections();
 
 // Regular global variables, only visible in this module.
+var unsalientCollections = new VRECollections(); // not currently selected
```

Comment:
Not really clear to me what this means; is this all collections except the one that is currently shown (if any)?

Reply:
I understand now; perhaps change comment to "all collections except currently selected one"?